### PR TITLE
Feature - history order

### PR DESCRIPTION
**Pull request by at-datphung**

### CODE REVIEW

File: MyApp/View/Controllers/HistoryOrder/HistoryOrderViewController.swift [hunk 1]
```diff
@@ -9,35 +9,71 @@
...
 import MVVM
 
-class HistoryOrderViewController: UIViewController {
+class HistoryOrderViewController: ViewController {
     @IBOutlet weak var historyOrderTableView: UITableView!
 
     var viewModel = HistoryOrderViewModel()
 
     override func viewDidLoad() {
         super.viewDidLoad()
```
Comment (at-daiho) [inline MyApp/View/Controllers/HistoryOrder/HistoryOrderViewController.swift]: WARNING - Lines should not have trailing whitespace.

**Comment by at-daiho** [comment]
Total linter violations in pull request: 1
Serious: 0
Warnings: 1

**Comment by at-daiho** [comment]
:+1: There are no linter violations.

**Comment by at-hieutran** [comment]
LGTM

**Pull request merged**